diff --git a/dist/react-error-boundary.js b/dist/react-error-boundary.js
index 50707fb1701cda610d9581bb8325047335c3e406..9cd93784e0725759f3c0f93d5214e4ea9bc863ba 100644
--- a/dist/react-error-boundary.js
+++ b/dist/react-error-boundary.js
@@ -1,3 +1,4 @@
+"use client";
 var $8zHUo$react = require("react");
 
 function $parcel$exportWildcard(dest, source) {
@@ -158,7 +159,6 @@ function $9e88dd86e0bb2944$export$f0c7a449e0cfaec7(component, errorBoundaryProps
 var $faefaad95e5fcca0$exports = {};
 
 
-"use client";
 $parcel$exportWildcard(module.exports, $6d6d4999e62b3ee0$exports);
 $parcel$exportWildcard(module.exports, $4a61716688322eb0$exports);
 $parcel$exportWildcard(module.exports, $3c4937b727a6fcfb$exports);
diff --git a/dist/react-error-boundary.module.js b/dist/react-error-boundary.module.js
index e33bc646a9fef2f0e7cce736f13d103a248bb279..c81dd8c4cadcb7b007a69333961a116ab550873f 100644
--- a/dist/react-error-boundary.module.js
+++ b/dist/react-error-boundary.module.js
@@ -1,3 +1,4 @@
+"use client";
 import {isValidElement as $hgUW1$isValidElement, createElement as $hgUW1$createElement, Component as $hgUW1$Component, createContext as $hgUW1$createContext, useContext as $hgUW1$useContext, useState as $hgUW1$useState, useMemo as $hgUW1$useMemo, forwardRef as $hgUW1$forwardRef} from "react";
 
 function $parcel$export(e, n, v, s) {
@@ -142,8 +143,6 @@ function $62ff477d53f02a5b$export$f0c7a449e0cfaec7(component, errorBoundaryProps
 var $81c1b644006d48ec$exports = {};
 
 
-"use client";
-
 
 export {$44d7e150ebc754d2$export$e926676385687eaf as ErrorBoundary, $ebb31c7feaa4405e$export$b16d9fb1a22de840 as ErrorBoundaryContext, $7c3c25b3f398a9d6$export$c052f6604b7d51fe as useErrorBoundary, $62ff477d53f02a5b$export$f0c7a449e0cfaec7 as withErrorBoundary};
 //# sourceMappingURL=react-error-boundary.module.js.map